python/tests/graph/test_import.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import, division, print_function
+
+import contextlib
+import shutil
+import numpy as np
+import os
+import tensorflow as tf
+import tempfile
+import glob
+
+import sparkdl.graph.utils as tfx
+from sparkdl.graph.input import TFInputGraph
+
+
+class TestGraphImport(object):
+    def test_graph_novar(self):
+        gin = _build_graph_input(lambda session:
+            TFInputGraph.fromGraph(session.graph, session, [_tensor_input_name],
+                                                        [_tensor_output_name]))
+        _check_input_novar(gin)
+
+    def test_graphdef_novar(self):
+        gin = _build_graph_input(lambda session:
+                                 TFInputGraph.fromGraphDef(session.graph.as_graph_def(),
+                                                           [_tensor_input_name], [_tensor_output_name]))
+        _check_input_novar(gin)
+
+    def test_saved_model_novar(self):
```

Comment:
(Context: design)
One issue with the current design is that, in order to test two types of graphs, we essentially doubled the amount of code (and actually missed `test_graphdef_var`, `test_graph_var`, etc.).